tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -782,10 +752,9 @@ def test_work_order_with_no_indata (self):
             self.test_obj.build_request_output['request_obj'])
 
         assert (
-                verify_test(
-                    result_response, 0,
-                    self.test_obj.build_request_output['pre_test_output'],
-                    self.test_obj.build_request_output['action_obj'])
+                check_negative_test_responses(
```

Comment:
Similar to above.

Reply:
This is the right change to handle the error. earlier it was verify test and now the issue has been fixed and it gives the result as "Missing parameter inData"